cypress/features/sanity/sanity.feature

```diff
@@ -13,6 +15,8 @@ Feature: Sanity tests
   @sanity
   Scenario Outline: Component <component> page and open preview
     When I open "<component>" component page
+      And click into Accessibility Tab
+      And click reRun tests button
```

Comment:
one step, no click word

Reply:
I leaved two steps, but changed the name of steps